<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetBaseInfo</class>
 <widget class="QDialog" name="GetBaseInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="m_btnStart">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btnCancel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>440</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>201</width>
     <height>39</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Sys MemUsage：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_TextSysMemUsage">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>201</width>
     <height>39</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Sys PhysMem：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_TextSysPhysMem">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>201</width>
     <height>39</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Sys ViruMem：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_TextSysVirtMem">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>201</width>
     <height>39</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Sys AvailPhysMem：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_TextAvailPhysMem">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>201</width>
     <height>39</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Sys AvailVirtsMem：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_TextAvailVirtMem">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_6" native="true">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>201</width>
     <height>39</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>当前程序的私有内存:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_TextAppPrivateMem">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>m_btnStart</zorder>
  <zorder>m_btnCancel</zorder>
  <zorder>widget</zorder>
  <zorder>widget_2</zorder>
  <zorder>widget_3</zorder>
  <zorder>widget_4</zorder>
  <zorder>widget_5</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>widget_6</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
